### PR TITLE
Feature/description support fields length 1000

### PR DESCRIPTION
**Pull request by McFlat**
This is the client/server part of pull request #1262 which was only the migration, but the migration is included in this pull request so I closed the other pull request(1262)

### CODE REVIEW

**Comment by Jorropo** [comment]
What is the performance impact ?
(How many it slow down peertube) ?

File: server/initializers/migrations/0280-description-support.ts [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+      defaultValue: null
+    }
+    await utils.queryInterface.changeColumn('videoChannel', 'description', data)
+  }
+
+  {
+    const data = {
+      type: Sequelize.STRING(1000),
+      allowNull: true,
+      defaultValue: false
```
Comment (Chocobozzz) [inline server/initializers/migrations/0280-description-support.ts]: null

Comment (McFlat) [reply server/initializers/migrations/0280-description-support.ts]: ah yes, you're right

Comment (McFlat) [reply server/initializers/migrations/0280-description-support.ts]: I've also increased the video support side on the client side, thanks for pointing that out

File: client/src/app/shared/forms/form-validators/video-validators.service.ts [hunk 2]
```diff
@@ -79,7 +79,7 @@ export class VideoValidatorsService {
     }
 
     this.VIDEO_SUPPORT = {
-      VALIDATORS: [ Validators.minLength(3), Validators.maxLength(500) ],
+      VALIDATORS: [ Validators.minLength(3), Validators.maxLength(1000) ],
       MESSAGES: {
         'minlength': this.i18n('Video support must be at least 3 characters long.'),
         'maxlength': this.i18n('Video support cannot be more than 500 characters long.')
```
Comment (Chocobozzz) [inline client/src/app/shared/forms/form-validators/video-validators.service.ts]: 1000 :)

Comment (rigelk) [inline client/src/app/shared/forms/form-validators/video-validators.service.ts]:
```suggestion
         'maxlength': this.i18n('Video support cannot be more than 1000 characters long.')
```

Comment (McFlat) [reply client/src/app/shared/forms/form-validators/video-validators.service.ts]: I've made the fix, not sure where my focus went haha

**Comment by Chocobozzz** [comment]
Thanks @McFlat

**Pull request merged**